packages/docs/src/components/documentation.tsx

```diff
@@ -166,6 +169,19 @@ export class Documentation extends React.PureComponent<IDocumentationProps, IDoc
         Utils.safeInvoke(this.props.onComponentUpdate, activePageId);
     }
 
+    private updateTitle(){
```

Comment:
Missing space before the brace: `private updateTitle() {`.

This will fail our build due to [`prettier`](https://prettier.io/) formatting checks. Use VS Code with the TSLint extension with `autoFixOnSave=true`, or run `yarn lint-fix` from the root directory to auto-fix formatting stuff. ([Read our editor docs](https://github.com/palantir/blueprint/wiki/Editor-integration))